recovery.md

```diff
@@ -0,0 +1,82 @@
+# The recovery partition
+
+On Ubuntu Core 20 systems there is a new system-recovery partition. It
+is big enough to contain a certain number of recovery "systems" that
+can then be used to recover or reinstall a broken system.
+
+A full "seed" to reinstall on a generic amd64 system is roughly 280MB
```

Comment:
do we how big things like network manager etc are?  just 20mb for those seems tight?

Reply:
I increased this now.